### PR TITLE
Big pile of small fixes

### LINKED ISSUE #58: Use new smut orc mechanic
Change smut orc code to use the new cold mechanic to get bridge pieces

### LINKED ISSUE #104: Does not go to Trapper to complete Quest
After defeating the yeti it does not go back to the Trapper to complete quest

### LINKED ISSUE #108: Select the right votes in Dark Gyffe
On voting options do not select anything with familiars and always select the +HP item

### LINKED ISSUE #134: Was informed to report fighting a crate
The config file says to report if I fight a crate. Unsure what to attach however. Step to.....reproduce what I did, no guarantee this will reproduce bug

1. Svn installed script for first time into mafia
1. Ran the gcli command to initiate it
1. Script ran successfully for ~120 turns
1. Manually stopped to look into config file/page in relay browser
1. Adjusted many options (too many to manually type. Happy to screen shot or whatever)
1. Started script again
1. Ended up in noob cave

Note, please close or delete this issue if it is not helpful, only reporting because script told me to

======EDIT===========EDIT=============EDIT===============

As it turns out I did not save the config file. So updated steps below

1. Svn installed script for first time into mafia
1. Ran the gcli command to initiate it
1. Script ran successfully for ~120 turns
1. Manually stopped to look into config file/page in relay browser
1. Adjusted many options (did not save file however)
1. Started script again
1. Ended up in noob cave

### LINKED ISSUE #180: Cast mist form in Twin Peak or Mountain climbing in DG
There is still room 237 and ninja lair to do, but the script did not know how to achieve +4 stench and +5 cold resistance, so went on to powerleveling (see the following log). 

Maybe add a mechanism that cast mist form when needing elemental resistance for quest in DG? 

Post Adventure done, beep.
Turn(290): Starting with 237 left and 0 pulls left at Level: 9
Encounter: -15.0 Exp Bonus: 19.619047619047617
Meat Drop: 100.0 Item Drop: 100.0
HP: 132/132, MP: 329/345
Tummy: 5/5 Liver: 4/4 Spleen: 0/15
ML: 76 control: 11
Delay between adventures... beep boop... 
Countdown: 1 second...
Waiting completed.
Putting on ghost of a necklace...
Equipment changed.
Retrieving quest data...
Maximizing...
64 combinations checked, best score 1.00
A-Boo Peak: 90
Maximizing...
128 combinations checked, best score 5.00
Current HP: 132/132
Expected cold damage: 239 Expected spooky damage: 139
Expected Cold Resist: 5 Expected Spooky Resist: 9 Expected HP Difference: 0
Nevermind, that peak is too scary!
I seem to need to power level, or something... waaaa.
Countdown: 10 seconds...
KoLmafia declares world peace.

### LINKED ISSUE #189: Setting choiceAdventure values
I tried sl_ascend on a Dark Gyffte run today and noticed that it wasn't taking advantage of the Death Rattlin' choiceAdventure.

```
Encounter: Death Rattlin'
Could not parse: You lose interest in the drawers and wander off
```
On checking the choiceAdventure value it was set to skip.  Is setting choiceAdventures outside the scope of the script?

### PR DESCRIPTION
**Pull request by jaspercb**
Closes #189 (Blech House NCs were previously only being set in HC)
Closes #180 (use mist form for room 237)
And some bookkeeping:
Closes #108 (this was fixed upstream)
Closes #104 (I... think this is fixed by our paranoid refreshing)
Closes #58 ("use new smut orc mechanic", there's room for improvement but it's more or less fixed)
Closes #134 ("fight a crate", not reproducible)

### CODE REVIEW

File: RELEASE/scripts/sl_ascend.ash [hunk 1]
```diff
@@ -14685,7 +14685,9 @@ void print_help_text()
 {
 	print_html("Thank you for using sl_ascend!");
 	print_html("If you need to configure or interrupt the script, choose <b>soolascend</b> from the drop-down \"run script\" menu in your browser.");
-	print_html("If you want to contribute, please open an issue at <a href=\"https://github.com/soolar/sl_ascend/issues\">on Github</a>");
+	print_html("If you want to contribute, please open an issue <a href=\"https://github.com/soolar/sl_ascend/issues\">on Github</a>");
+	print_html("A FAQ with common issues (and tips for a great bug report) <a href=\"https://docs.google.com/document/d/1AfyKDHSDl-fogGSeNXTwbC6A06BG-gTkXUAdUta9_Ns\">can be found here</a>");
+	print_html("The developers also hang around on the Ascension Speed Society discord.");
```
Comment (soolar) [inline RELEASE/scripts/sl_ascend.ash]: Can you add the same link to the discord as is in README.md to this too? Might as well.

File: RELEASE/scripts/sl_ascend.ash [hunk 2]
```diff
@@ -11364,16 +11368,8 @@ boolean LX_handleSpookyravenFirstFloor()
...
-		{
-			equip($slot[weapon], $item[Pool Cue]);
-		}
-		if(have_equipped($item[Pool Cue]))
-		{
-			expectPool += 3;
-		}
 
-		if(!possessEquipment($item[Pool Cue]))
+		if(!possessEquipment($item[Pool Cue]) && !posessEquipment($item[7964]) && !posessEquipment($item[2268]) && !posessEquipment($item[7961]))
```
Comment (soolar) [inline RELEASE/scripts/sl_ascend.ash]: Can you add inline comments saying what the items referred to by number are, just for easier future reference?

File: RELEASE/scripts/sl_ascend.ash [hunk 3]
```diff
@@ -11369,7 +11366,7 @@ boolean LX_handleSpookyravenFirstFloor()
 			expectPool += 3;
 		}
 
-		if(!possessEquipment($item[Pool Cue]) && !posessEquipment($item[7964]) && !posessEquipment($item[2268]) && !posessEquipment($item[7961]))
+		if(!possessEquipment($item[Pool Cue]) && !possessEquipment($item[7964]) && !possessEquipment($item[2268]) && !possessEquipment($item[7961]))
```
Comment (soolar) [inline RELEASE/scripts/sl_ascend.ash]: Maybe try doing `ashq import<sl_ascend.ash>` before committing next time 😛 

Or better yet (I just did this myself) `alias testsl => ashq import<sl_ascend.ash> print("All is well.")` and then just `testsl` before committing. Only good for catching compilation errors, but better than nothing.

File: RELEASE/scripts/sl_ascend.ash (hunk 2 above)
Comment (jaspercb) [reply RELEASE/scripts/sl_ascend.ash]: I can do one better, and write self-documenting code

File: RELEASE/scripts/sl_ascend.ash (hunk 3 above)
Comment (jaspercb) [reply RELEASE/scripts/sl_ascend.ash]: :(

File: RELEASE/scripts/sl_ascend/sl_ascend_header.ash [hunk 4]
```diff
@@ -613,7 +614,7 @@ int kgb_tabCount(string page);								//Defined in sl_ascend/sl_mr2017.ash
 boolean kgb_getMartini();									//Defined in sl_ascend/sl_mr2017.ash
 boolean kgb_getMartini(string page);						//Defined in sl_ascend/sl_mr2017.ash
 boolean kgb_getMartini(string page, boolean dontCare);		//Defined in sl_ascend/sl_mr2017.ash
-boolean kgbModifiers(string mod);							//Defined in sl_ascend/sl_mr2017.ash
+boolean kgbModifiers(string mod);							//Defined in sl_ajkscend/sl_mr2017.ash
```
Comment (soolar) [inline RELEASE/scripts/sl_ascend/sl_ascend_header.ash]: minor oops here

**Pull request merged**